tools/serverpod_cli/lib/src/commands/generate.dart

```diff
@@ -25,12 +25,21 @@ class GenerateCommand extends ServerpodCommand {
       negatable: false,
       help: 'Watch for changes and continuously generate code.',
     );
+    argParser.addFlag(
+      'open-api',
+      abbr: 'o',
+      defaultsTo: false,
+      negatable: false,
+      help: 'Generate open-api schema.',
+    );
   }
 
   @override
   Future<void> run() async {
     // Always do a full generate.
     bool watch = argResults!['watch'];
+    // Whether to generate an OpenAPI schema or not
+    bool genOpenApi = argResults!['open-api'];
```

Comment:
It may be better to use a bit more descriptive name, like the full `generateOpenApi`.